rioxarray/rioxarray.py

```diff
@@ -238,6 +255,8 @@ def set_crs(self, input_crs, inplace=True):
         ----------
         input_crs: object
             Anythong accepted by `rasterio.crs.CRS.from_user_input`.
```

Comment:
Not part of the diff, but there's a typo here.

Reply:
Nice catch.